### PR TITLE
NER Parameter Tuning

### PR DESCRIPTION
**Pull request by cowchipkid**
This collection of scripts in total does a parameter sweep over learning rate and thickness.

### CODE REVIEW

**Comment by danyaljj** [comment]
Do we want to checkin `ner/scripts/paramsweep/run.jar` into the repo?

**Comment by cowchipkid** [comment]
Yes. I no longer have the source that little app I wrote.

**Comment by mssammon** [comment]
I'd prefer non-binary -- even if it were just decompiled and cleaned up a bit.... how much work would that be?

**Comment by cowchipkid** [comment]
It would be faster to rewrite it. Might be able to do this in bash, but I am not that well versed bash… If we want to put that much labor into, I would also suggest we change it such that it no longer programmatically modifies and recompiles the LBJ file, but rather simply modifies the config files, adding the appropriate parameters there.

**Comment by mssammon** [comment]
any idea how much time it would take (in Java)?

**Comment by cowchipkid** [comment]
1 - 2 hrs to write, but then would need to be tested, but I would be hands off at that point.

**Comment by mssammon** [comment]
definitely worth rewriting, then.  Please go ahead and add the code to a suitable package in the cogcomp-nlp project.

**Comment by cowchipkid** [comment]
Moving the illinois-cogcomp-nlp to cogcomp-nlp has broken the scripts anyway. I will fix those as well.

File: ner/src/main/java/edu/illinois/cs/cogcomp/ner/ParameterSweep.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/**
```
Comment (mssammon) [inline ner/src/main/java/edu/illinois/cs/cogcomp/ner/ParameterSweep.java]: need to add license header -- `mvn license:format`

**Pull request merged**